packages/shared/src/hooks/index.ts

```diff
@@ -29,3 +29,4 @@ export * from './useProgressAnimation';
 export * from './feed';
 export * from './useOnboardingChecklist';
 export * from './useBookmarkProvider';
+export * from './useInView';
```

Comment:
Let's avoid barrel export, its bad for tree shaking, so we need to evaluate with rest of the web team which way we want to go.

Reply:
Ah think indeed besides icons we said to not use it until we chat about it further.
(but no hard requirement for me)

Reply:
yeah, non blocking, tree shaking does not work either way :hidethepain: